clients/fides-js/src/components/ConsentButtons.tsx

```diff
@@ -75,6 +59,28 @@ export const ConsentButtons = ({
         </Fragment>
       )}
     </div>
+    <div
+      className={`${
+        isInModal
+          ? "fides-modal-button-group fides-modal-secondary-actions"
+          : "fides-banner-button-group fides-banner-secondary-actions"
+      } ${
+        i18n.availableLanguages?.length > 1 ? "fides-button-group-i18n" : ""
+      }`}
+    >
+      {i18n.availableLanguages?.length > 1 && (
```

Comment:
I noticed we're checking this condition 2 times. I would consider having a variable like "showLanguageSelector".